shuup/admin/static_src/base/js/tour.js

```diff
@@ -0,0 +1,192 @@
+            ],
+            attachTo: "li a[data-target-id='category-1'] right"
+        }, {
+            text: [
+                gettext("All your products and product options are in here, there’s a bunch. Manufacturers, suppliers, and your inventory and stock management are inside. You can add more products and categories here too.")
+            ],
+            attachTo: "li a[data-target-id='category-2'] right"
+        }, {
+            text: [
+                gettext("All your shopper data is inside. Check out your best customers, setup special user groups, and get info to target them so they wanna buy.")
+            ],
+            attachTo: "li a[data-target-id='category-3'] right"
+        }, {
+            text: [
+                gettext("Everyone likes reports :D There are a bunch of ways to set them up to get the data you need and to see how and where you are making or losing money.")
+            ],
+            attachTo: "li a[data-target-id='category-4'] right"
+        }, {
+            text: [
+                gettext("Here you can create sales, coupons, and marketing campaigns that entice your shoppers. Set’em up based on catalog or basket and make recommended product upsells and crossells. Or do something unique that will make’em buy.")
+            ],
+            attachTo: "li a[data-target-id='category-5'] right"
+        }, {
+            text: [
+                gettext("This is the place to customize your site and add pages, shops, and sales carousels. You can add media like photos and videos or completely change the look of your store. It’s super customizable to make it exactly how you want.")
+            ],
+            attachTo: "li a[data-target-id='category-6'] right"
+        }, {
+            text: [
+                gettext("This is your connection interface. Addons and other systems you use can be attached to your store through powerful data connections. Supercharge your site and gather crazy amounts of data with integrations to CRMs and ERPs, POS’s and PIM’s, or any other acronym you can think of.")
+            ],
+            attachTo: "li a[data-target-id='category-7'] right"
+        }, {
+            text: [
+                gettext("You know, options and stuff :D Its got everything from taxes and tax rules, payment and shipping methods, to notification settings. Tweak away.")
+            ],
+            attachTo: "li a[data-target-id='category-8'] right"
+        }, {
+            text: [
+                gettext("Find anything, and I mean anything that is inside your store. This includes settings, products, users, and addons.")
+            ],
+            attachTo: "#site-search bottom"
+        }, {
+            text: [
+                gettext("Peep your shop and all the cool changes you’ve made.")
+            ],
+            attachTo: ".shop-btn left"
```

Comment:
yea seems to be an issue in the third party lib. Anything near the edges of the screen is missing arrows.